distributed/cli/dask_ssh.py

```diff
@@ -50,9 +50,11 @@
 @click.option('--remote-dask-worker', default=None, type=str,
```

Comment:
Lets change the default here with `default="distributed.cli.dask_worker"`

Reply:
Done.  (But note that we're then setting the default remote worker in two locations: `cli.dask_ssh.main` and `deploy.SSHCluster.__init__`.